metadata/src/main/java/edu/unc/lib/dl/util/JobStatusFactory.java

```diff
@@ -79,64 +79,59 @@ public void clearStale(String depositUUID) {
 		uuids.addAll(queued);
 		uuids.addAll(working);
 
-		Jedis jedis = getJedisPool().getResource();
-		try {
+		try (Jedis jedis = getJedisPool().getResource()) {
 			for (String uuid : uuids) {
 				jedis.del(JOB_STATUS_PREFIX + uuid);
 				jedis.lrem(DEPOSIT_TO_JOBS_PREFIX + depositUUID, 0, uuid);
 			}
-		} finally {
-			getJedisPool().returnResource(jedis);
 		}
 	}
 
 	public void completed(String jobUUID) {
-		Jedis jedis = getJedisPool().getResource();
-		jedis.hset(JOB_STATUS_PREFIX + jobUUID,
-				JobField.status.name(), JobStatus.completed.name());
-		jedis.hset(JOB_STATUS_PREFIX + jobUUID,
-				JobField.endtime.name(),
-				String.valueOf(System.currentTimeMillis()));
-		getJedisPool().returnResource(jedis);
+		try (Jedis jedis = getJedisPool().getResource()) {
+			jedis.hset(JOB_STATUS_PREFIX + jobUUID,
```

Comment:
Actually sorry I was thinking of DepositStatusFactory.addManifest.  I think the indent is correct here though